src/Extensions/Extensions.jl

```diff
@@ -40,6 +40,18 @@ function Formats.metamacro(::META"set", body, mod, obj)
 end
 
 """
+!!summary(Set the value for a field in an object's metadata, and splice the content in place of the ``metamacro`` call.)
```

Comment:
Should the summary just read

```jl
"""
!!summary(Equivalent to ``!!set`` followed by ``!!get`` for the provided key.)
...
"""
```
Thoughts?

(We should probably have a ``!!related(get, set)`` metamacro that makes a linked "see also" section in the final metadata. Not too important right now though.)